design-document.md

```diff
@@ -95,6 +111,7 @@ an aggergation function F:
    and nothing else. 
 1. Robustness. The adversary can influence the output of F only by reporting false 
    (untruthful) data. The output cannot be influenced in any other way.
+1. Anonymity. The adversary cannot learn which client submitted which data value.
```

Comment:
The [2017 paper](https://crypto.stanford.edu/prio/paper.pdf) calls out privacy and anonymity as distinct security properties (section 2). Anonymity is the adversary not knowing which client submitted which input, privacy is the adversary not knowing what the inputs are. The reason I added anonymity to this list is because I think that in the setting where aggregators talk directly to clients, privacy is preserved but client anonymity is weakened since the aggregator can identify clients e.g. by IP. Further we can sketch out a mitigation -- an anonymizing proxy -- so I think it's worth discussing the anonymity property.